lib/pantry/handlers/alexa_request_handler.rb

```diff
@@ -0,0 +1,27 @@
+module Pantry
+
+  module Handlers
+
+    class AlexaRequestHandler
+
+      @@handlers = {
+        'INTENT_REQUEST' => 'Pantry::Handlers::IntentRequestHandler',
+        'LAUNCH_REQUEST' => 'Pantry::Handlers::LaunchRequestHandler',
+        'SESSION_ENDED_REQUEST' => 'Pantry::Handlers::SessionEndedRequestHandler'
+      }
```

Comment:
personally I would probably go with something like
```ruby
def handler_for(request_type)
  {
    'INTENT_REQUEST'        => Pantry::Handlers::IntentRequestHandler,
    'LAUNCH_REQUEST'        => Pantry::Handlers::LaunchRequestHandler,
    'SESSION_ENDED_REQUEST' => Pantry::Handlers::SessionEndedRequestHandler
  }[request_type]
end
```

avoid the need for `constantize`, and allow you a good place for handling the case of an invalid request type coming in. In general I rarely find a use for @@ class variables, either go with class/instance methods, or CONSTANTS.